Loader/ComboLoader.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * File containing the ComboLoader class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Loader;
+
+use eZ\Publish\Core\Base\Exceptions\NotFoundException;
+
+class ComboLoader implements Loader
```

Comment:
I'd like a bit of phpdoc about this implementation of the Loader interface, and what it does in particular.